server/events/project_command_builder_internal_test.go

```diff
@@ -382,6 +386,7 @@ workflows:
 			repoCfg: `
 version: 3
 automerge: true
+autodiscover: true
```

Comment:
so we can get more granular with `autodiscover`, it would be nice to have this format instead

```yaml
autodiscover:
  enabled: true
```

Reply:
Nice, Done man.